tests/integration/modules/test_linux_acl.py

```diff
@@ -5,22 +5,24 @@
 import os
 import shutil
 
+import pytest
+
 # Import Salt Testing libs
 from tests.support.runtests import RUNTIME_VARS
 from tests.support.case import ModuleCase
 from tests.support.mixins import AdaptedConfigurationTestCaseMixin
-from tests.support.helpers import skip_if_binaries_missing
 
 # Import salt libs
 import salt.utils.files
 import salt.utils.user
 
 
 # Acl package should be installed to test linux_acl module
-@skip_if_binaries_missing(['getfacl'])
+@pytest.mark.skip_if_binaries_missing('getfacl')
 # Doesn't work. Why?
-# @requires_salt_modules('acl')
-# @requires_salt_modules('linux_acl')
+# @pytest.mark.requires_salt_modules('acl')
+# @pytest.mark.requires_salt_modules('linux_acl')
```

Comment:
Though, after reviewing the rest of the code, I think either this is not clearly named and could be `requires_salt_function` - because everything else is like `requires_salt_modules('pkg.list_pkgs')`, which is different than `requires_salt_modules('pkg')` 🤷‍♂ 

I'm not familiar with that helper, though - just observations based on reviewing *this* code 😝

Reply:
Hm. Looking at that code, I'm not sure *why* this would not work.

If it still is broken, what I would do is just add `if 'acl' in required_salt_modules: raise SkipTest(...)` (same thing for `'linux_acl'`) in that function to make sure *that* works. That would tell us that the skip is correctly working.

Then if that part is working I'd start to look into why the sminion isn't producing the desired information here.

Reply:
This is a `sed`ed change.
I really don't want to address making this test work, or not in this PR.
Perhaps on a follow-up.